src/teleop/gui/gui.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import subprocess
+import rospy
+import os
+
+rospy.init_node("gui")
+
+os.chdir("../../../src/mrover/src/teleop/gui")
+bashCommand = "yarn serve"
+process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
```

Comment:
seems like it would be much better to change this entire file to a bash script: https://answers.ros.org/question/51474/can-i-run-a-bash-script-using-roslaunch/